pokemon-showdown

```diff
@@ -16,15 +18,18 @@ try {
 try {
 	require.resolve('sockjs');
 } catch (e) {
-	let command = 'npm install --production';
-	console.log('Installing dependencies: `' + command + '`...');
-	require('child_process').spawnSync('sh', ['-c', command], {stdio: 'inherit'});
+	console.log('Installing dependencies...');
+	child_process.spawnSync('npm', ['install', '--production'], {stdio: 'inherit'});
 }
 
-// start the server
+// Start the server
 
 let port;
 if (process.argv[2]) {
 	port = parseInt(process.argv[2]); // eslint-disable-line radix
```

Comment:
Do we still need the `eslint-disable-line` here? `radix` shouldn't even be turned on; `parseInt` defaults radix to `10` ever since ES5.